.prettierignore

```diff
@@ -5,3 +5,4 @@ android
 ignite/ignite.json
 package.json
 __generated__
+app/components/price-graph/data-tst.ts
```

Comment:
This file causes prettier to take much longer to run.  It's just a massive json blob so I've ignored it.